eng/pipelines/runtime-official.yml

```diff
@@ -41,13 +41,12 @@ extends:
       # Localization build
       #
 
-      # disabled due to https://github.com/dotnet/runtime/issues/90466
```

Comment:
Not sure if this is still broken or not.  Make sure you are able to test this PR before merging.  I don't think your PR validation build will test it.

Reply:
Maybe there's an azp run command to run it. Should I re-enable that in a separate PR?

Reply:
Ah nevermind, your other comment suggesting adding the parameter is precisely the lines I uncommented.

Reply:
This still won't run in PR validation. It only runs for official builds.  Also the branch filter might be getting in the way regardless. Might be hard to test this.